Aquarius/Src/Aquarius/Sound/SoundBuffer.cpp

```diff
@@ -74,18 +75,6 @@ namespace Aquarius {
             return buffer;
         }
 
-        SoundBuffer::SoundBuffer()
-        {
-            m_EffectBuffer.clear();
-        }
-
-        SoundBuffer::~SoundBuffer()
-        {
-            alDeleteBuffers(m_EffectBuffer.size(), m_EffectBuffer.data());
```

Comment:
I think you still might want this to exist, because if it is anything like OpenGL then this will not be handled by the unique pointer destructor